### DISCUSSION TITLE
Database seems to delete itself it mount is changed

### DISCUSSION THREAD

**Discussion by Nightwalker83** [dani-garcia/vaultwarden#5006] [Q&A]
Hi,

I am experiencing a problem in where the vaultwarden database that I have add passwords to keeps getting deleted. The deletion occurs when I try to change the mount point. However, the problem seems to be with the files on the actual drive rather than the container itself amd I can not figure out why.

Thanks,

Aaron

**Comment by Nightwalker83** [comment]
Seems the database disappears when using Bitwarden too
https://community.bitwarden.com/t/warning-all-data-completely-deleted-from-account/50444/28?page=1

**Comment by stefan0xC** [comment]
> change the mount point

What do you mean by that?

**Reply by stefan0xC** [reply]
If your [persistent data](https://github.com/dani-garcia/vaultwarden/wiki/Backing-up-your-vault#backing-up-data) (including the `db.sqlite3`) is stored in `/vw-data` that is mounted to `/data` in the container do you change it to e.g. `/var/lib/vaultwarden` or do you mount `/vw-data` to something else in the container? The first one would obviously not work as it would be like a new installation (unless you also moved the files to the new installation). And the latter would require you to update the `DATA_FOLDER` configuration option so that Vaultwarden knows where to look for the persistent data.

**Comment by Nightwalker83** [comment]
I had docker on a different drive and every I changed the mount point between /mnt and /media for reason it would not detect the existing database.  So my path would of been ./docker/volumes/vaultwarden

**Reply by stefan0xC** [reply]
Not sure I understand your setup. So the docker volumes are on a separate drive? Why would you change the mount point? Can't you bind mount the path to a fixed location?

**Reply by Nightwalker83** [reply]
Yes, I can! I try to mount them to /mnt/location but Ubuntu puts the drive at /media/location by default. Maybe I should keep mount to ./ instead of trying to use an absolute path.
